src/Resources/config/services.yml

```diff
@@ -0,0 +1,5 @@
+services:
+    contao_newsletter.data_container.member:
```

Comment:
Still hooks and callbacks are technically events. And we do not (yet) have DCA handlers in the core, do we?